crates/oxc_ast/src/ast_impl/js.rs

```diff
@@ -293,6 +293,22 @@ impl<'a> Expression<'a> {
         }
     }
 
+    /// Returns the identifier or the assignment target's identifier of the given expression.
+    pub fn extract_id_or_assign_to_id(&self) -> Option<&IdentifierReference<'a>> {
```

Comment:
I think we should dump these util methods into the minifier unless it's super generic.

Reply:
I was wondering where to put this. I moved it in the minifier.